src/cmd/flux-jobs.py

```diff
@@ -93,7 +93,8 @@ def __init__(self, info_resp):
         d.update(info_resp)
 
         #  Rename "state" to "state_id" until returned state is a string:
-        d["state_id"] = d.pop("state")
+        if d.has_key("state"):
```

Comment:
(oops forgot to hit add comment)

Looks like `has_key` method is no longer available in Python 3. This [guide](https://portingguide.readthedocs.io/en/latest/dicts.html) suggests using `in` instead.

Reply:
oh lol, you posted just before I did :P